### DISCUSSION TITLE
Get all starred gist of a user and public gists with graphql api

### DISCUSSION THREAD

**Discussion by Arjun-sna** [community/community#24450] [Apps, API and Webhooks]
<p>Currently, I was able to fetch a user’s public and private Gists using the Graphql API. But I couldn’t find a way to fetch all public Gists and also cannot get the list of Gists starred by a user.</p>
<p>Basically, there is no equivalent Graphql API for below REST APIs</p>
<pre><code>/gists/public

/gists/starred
</code></pre>

**Comment by lee-dohm** [comment, marked as answer]
<p>There are some things that you still can’t do with the GraphQL v4 API that are possible with the REST v3 API. For now, you may still have to use REST to access some information. Our GraphQL API is being constantly updated however, so you may want to keep an eye on <a href="https://developer.github.com/v4/changelog/" rel="nofollow noopener">the GraphQL Schema changelog</a> for updates.</p>
<p>I hope that helps!</p>
